src/components/ContactInfo/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@bcrs-shared-components/contact-info",
+  "version": "1.0.0",
```

Comment:
I recommend "0.0.0" so that when Lerna first versions it, then you can select "1.0.0".

Reply:
oh nice, ok will do!